Ontology/Capability/Capability.json

```diff
@@ -14,6 +14,124 @@
       "schema": "dateTime",
       "writable": true
     },
+    {
+      "@type": "Property",
+      "description": {
+        "en": "The (optional) substance that a capability measures or interacts with; e.g., a TemperatureSensor can measure air temperature or water temperature, a Flow Demand Setpoint can be configured to control steam demand or chilled water demand, etc."
+      },
+      "displayName": {
+        "en": "substance"
+      },
+      "name": "substance",
+      "dtmi:dtdl:property:schema;2": {
+        "@type": "Enum",
+        "dtmi:dtdl:property:enumValues;2": [
+          {
+            "enumValue": "Water",
+            "name": "Water"
+          },
+          {
+            "enumValue": "Steam",
+            "name": "Steam"
+          },
+          {
+            "enumValue": "OutsideAir",
+            "name": "OutsideAir"
+          },
+          {
+            "enumValue": "MediumTemperatureHotWater",
+            "name": "MediumTemperatureHotWater"
+          },
+          {
+            "enumValue": "HotWater",
+            "name": "HotWater"
+          },
+          {
+            "enumValue": "HighTemperatureHotWater",
+            "name": "HighTemperatureHotWater"
+          },
+          {
+            "enumValue": "DomesticHotWater",
+            "name": "DomesticHotWater"
+          },
+          {
+            "enumValue": "DeionisedWater",
+            "name": "DeionisedWater"
+          },
+          {
+            "enumValue": "ChilledWater",
+            "name": "ChilledWater"
+          },
+          {
+            "enumValue": "Air",
+            "name": "Air"
+          }
+        ],
+        "valueSchema": "string"
+      },
+      "writable": true
+    },
+    {
+      "@type": "Property",
+      "description": {
+        "en": "The (optional) capability position relative to the system or space that it interacts with. E.g., a water temperature sensor can be positioned on the supply or return flow, air humidity can be measured entering or leaving a system, etc."
+      },
+      "displayName": {
+        "en": "position"
+      },
+      "name": "position",
+      "dtmi:dtdl:property:schema;2": {
```

Comment:
I see more values in the same excel, you just missed them or don't agree with the list? Thanks

Reply:
See previous comment.